pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -119,25 +119,25 @@ func syncOpenShiftAPIServer_v311_00_to_latest(c OpenShiftAPIServerOperator, orig
 	}
 
 	// manage status
-	var availableConditionReason string
+	var availableConditionReasons []string
 	var availableConditionMessages []string
 
 	switch {
 	case actualDaemonSet == nil:
-		availableConditionReason = "NoDaemon"
+		availableConditionReasons = append(availableConditionReasons, "NoDaemon")
```

Comment:
messy. We should have a condition for each one.  I think this good enough to backport, but in 4.3, they should be individual and we should clear the existing "Available".